docs/console/design-your-projects/custom-resources.md

```diff
@@ -4,16 +4,20 @@ title: Create Custom Resources
 sidebar_label: Create Custom Resources
 ---
 
-## How to use the Custom Resources
+## What is a Custom Resource?
 
-A Custom Resource allows you to define custom objects that are not part of the standard Console supported resources.
+A Custom Resource allows you to define custom objects that are not part of the standard Console resources and can be used to extend the Console capabilities. It provides a way to write configurations that can be translated into different configuration-as-code elements.
 
-For example, it is possible to:
+## How to use the Custom Resource
 
-- configure Kubernetes Custom Resource that are managed by the cluster (e.g. the Traefik `IngressRoute`);
+With Custom Resources it is possible to:
+
+- configure Kubernetes Custom Resources that are managed by the cluster (e.g. the Traefik `IngressRoute`);
 - generate manifests for different runtimes using the [External Configuration Generator](/console/company-configuration/providers/extensions/orchestrator-generator/overview.mdx).
```

Comment:
this is intended as something you could do but does not take into account "template-based" CR. we should add a new list item here telling that you can configure resource for any runtime using templates that are generated at deploy time (or something along this line)

Reply:
done